framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers/Pagination/PagerModel.cs

```diff
@@ -38,6 +38,11 @@ public PagerModel(long totalCount, int shownItemsCount, int currentPage, int pag
             PageSize = pageSize;
             TotalPageCount = (int)Math.Ceiling(Convert.ToDouble((decimal)TotalItemsCount / PageSize));
             Sort = sort;
+
+            if (pageUrl.IsNullOrWhiteSpace() || !pageUrl.StartsWith("/"))
```

Comment:
`pageUrl = "/" + pageUrl;` will error when `pageUrl` is `null`

Reply:
See: https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#null-strings-and-empty-strings
These overloads of the binary + operator perform string concatenation. If an operand of string concatenation is null, an empty string is substituted.